Refreshing miscellany (#144)
AlexsLemonade/training-specific-template: software-setup/rstudio-login.md

In other words, you will be programming in R via this server, and _you do not need to download R or RStudio for this workshop._

In this tutorial, we will get you set up with our server.

> In our [RStudio guide](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/00a-rstudio_guide.Rmd) and our [`intro-to-R-tidyverse` module](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse) we will work on becoming more comfortable with R and RStudio for programming.

Reviewer: These links feel a bit redundant? Either way.

```suggestion
In our [`intro-to-R-tidyverse` module](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse) we will work on becoming more comfortable with R and RStudio for programming.
```

Author: I updated this phrasing a bit actually since we don't actually teach that module for all workshops (looking at you, scAdvanced). Mostly I was bothered by the "we will work on," since maybe we won't!

What do you think of something like this?
```suggestion
Our [`intro-to-R-tidyverse` module](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse) contains lessons and resources for becoming more comfortable with R and RStudio for programming.
```

Reviewer: Sounds good to me!
